### PR TITLE
Update 04-data-structures-part1.Rmd

### PR DESCRIPTION
**Pull request by shu251**
Suggested modification for data frame explanation.

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _episodes_rmd/04-data-structures-part1.Rmd [hunk 1]
```diff
@@ -153,23 +153,27 @@ we did on them before, we run into trouble:
...
 
-We can see that it is a *data.frame* by calling the `class` function on it:
+When R reads a csv file, it reads it in as a *data frame*. Thus, when we loaded the `cats`
+csv file, it is stored as a data frame. We can check this by using the function `class()`.
 
 ```{r}
 class(cats)
 ```
+*Data frames* are composed of rows and columns, where each column has the 
+same number of rows. Different columsn in a data frame can be made up of different
```
Comment (Alanocallaghan) [inline _episodes_rmd/04-data-structures-part1.Rmd:170]: "columsn" -> "columns"

Comment (jcoliver) [reply _episodes_rmd/04-data-structures-part1.Rmd:170]: Thanks, @Alanocallaghan

**Comment by jcoliver** [comment]
Thanks, @shu251 ! Would you mind updating the PR with the typo correction @Alanocallaghan added?

File: _episodes_rmd/04-data-structures-part1.Rmd (hunk 1 above)
Comment (shu251) [reply _episodes_rmd/04-data-structures-part1.Rmd]: thanks @Alanocallaghan !

**Review by shu251 — COMMENTED** [review]
Resolved typo on line 170

**Comment by jcoliver** [comment]
Thanks again, @shu251 and @Alanocallaghan !

**Pull request merged**